### PR TITLE
[FIX] Chrome doesn't load additional search results when bottom is reached

### LINKED ISSUE #14706: Chrome doesn't load additional search results when bottom is reached
### Description:

When using the search function of Rocket.Chat in Chrome, it doesn't seem to load additional search results when the bottom of the view is reached and the endless spinner should actually show up to load more.

### Steps to reproduce:

1. Go to our open instance (https://open.rocket.chat)
2. Join "#support" room
3. Search for `from:frdmn`
4. Notice how it stops after showing messages from "20th February 2019"

### Actual behavior:

demo_bug

### Expected behavior:

On Electron, the spinner shows up and loads additional results (at least including "4th Feburary 2019":

demo_bug_working

### Server Setup Information:

- Version of Rocket.Chat Server: 1.1.0
- Operating System: Cloud
- Deployment Method: Cloud
- Number of Running Instances: Cloud
- DB Replicaset Oplog: Cloud
- NodeJS Version: Cloud
- MongoDB Version: Cloud

### Additional information

- Chrome Version: 74.0.3729.169 (official build) (64-bit)

Apparently this also happens in Firefox.

### PR DESCRIPTION
**Pull request by ggazzo**
Closes #14706

### CODE REVIEW

File: app/search/client/provider/result.js [hunk 1]
```diff
@@ -50,11 +64,6 @@ Template.DefaultSearchResultTemplate.onCreated(function() {
 	this.data.parentPayload.searchAll = this.globalSearchEnabled;
 
 	this.hasMore = new ReactiveVar(true);
```
Comment (tassoevan) [inline app/search/client/provider/result.js]: I couldn't find where this variable is set.

File: app/ui-master/client/loading/loading.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	text-align: center;
+	align-items: center;
+	justify-content: center;
+}
+
+.loading-animation > .bounce {
+	display: inline-block;
+
+	width: 10px;
+	height: 10px;
```
Comment (tassoevan) [inline app/ui-master/client/loading/loading.css]: Is it possible to make this size relative to the parent's height?

Comment (ggazzo) [reply app/ui-master/client/loading/loading.css]: <3 in the future ;)

**Review by tassoevan — DISMISSED** [review]

**Comment by Abhi0229** [comment]
Hey! I'd like to work on this issue.
I'll reproduce the issue locally, identify the root cause, and submit a PR with a tested fix.

**Pull request merged**